sql/2020/22_HTTP_2/measure_number_of_tcp_connections_per_site.sql

```diff
@@ -0,0 +1,41 @@
+# standardSQL
+# Measure number of TCP Connections per site.
+SELECT 
+  "mobile" AS client,
+  JSON_EXTRACT_SCALAR(payload, "$._protocol") AS protocol,
+  COUNT(0) AS num_pages,
+  APPROX_QUANTILES(_connections, 100)[SAFE_ORDINAL(50)] AS median,
+  APPROX_QUANTILES(_connections, 100)[SAFE_ORDINAL(75)] AS p75,
+  APPROX_QUANTILES(_connections, 100)[SAFE_ORDINAL(95)] AS p95
+FROM 
+  `httparchive.requests.2020_08_01_mobile` AS requests
+INNER JOIN
+  `httparchive.summary_pages.2020_08_01_mobile` AS summary
```

Comment:
Can we use the `httparchive.almanac.requests` table and run this on one go? Seems a bit inconsistent to use that in some places and these tables in others.

Reply:
Makes sense. Let me rewrite the query to make sure I get the same output